tests/plugins/a11yhelp/manual/commands.md

```diff
@@ -0,0 +1,52 @@
+@bender-tags: find, tc, 4.7.1, 16980
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea,sourcearea,htmlwriter,entities,toolbar,elementspath,undo,clipboard,format,basicstyles,autolink,autoembed,link,a11yhelp,pastetext
```

Comment:
You could reduce plugins count to simply `@bender-ckeditor-plugins: wysiwygarea,a11yhelp,pastetext`.

You want to be sure that there's as little noise as possible. That was one of tensions raised during testing phase 😉